catalog/transformer/catalog-transformer-xml/pom.xml

```diff
@@ -107,6 +111,7 @@
                             catalog-transformer-xml-binding,
                             jaxb2-basics-runtime,
                             gml-v_3_1_1-schema;inline=true,
+                            commons-lang3,
```

Comment:
Do we really need the embed or can we use `escapeXml()` from 2.6?

Reply:
I don't believe the escapeXml is what we want according to the docs:
https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringEscapeUtils.html#escapeXml(java.lang.String)

It only supports the 5 basic entities, similiar to what ours did previously.